signalfx-org-metrics/metrics.yaml

```diff
@@ -1,156 +1,283 @@
+    The entitlement for the number of containers for your subscription plan.
   
-      * Dimension(s): `orgId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.grossSpanBytesReceived
+        * Dimension(s): `orgId`
+        * Data resolution: 2 minutes
+  metric_type: gauge
+  title: sf.org.apm.subscription.containers
 
-sf.org.apm.grossSpanBytesReceivedByToken:
-  brief: The number of bytes Splunk APM received for a specific access token
-    from spans after decompression but before filtering and throttling.
+sf.org.apm.subscription.hosts:
+  brief: The entitlement for the number of hosts for your host-based subscription plan.
   description: |
-    The number of bytes Splunk APM receives for a specific access token
-    from spans after decompression but before filtering and throttling.
+    The entitlement for the number of hosts for your subscription plan.
   
-      * Dimension(s):  `orgId`, `tokenId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.grossSpanBytesReceivedByToken
+        * Dimension(s): `orgId`
+        * Data resolution: 2 minutes
+  metric_type: gauge
+  title: sf.org.apm.subscription.hosts
 
-sf.org.apm.numSpanBytesReceived:
-  brief: The number of bytes Splunk APM accepts for a span after decompression
-    after filtering and throttling.
+sf.org.apm.subscription.monitoringMetricSets:
+  brief: The entitlement for the number of Monitoring MetricSets as part of your subscription plan.
   description: |
-    The number of bytes Splunk APM accepts for a span after
-    decompression, filtering, and throttling.
-
-      * Dimension(s): `orgId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.numSpanBytesReceived
+    The entitlement for the number of Monitoring MetricSets as part of your subscription plan.
+  
+        * Dimension(s): `orgId`
+        * Data resolution: 2 minutes
+  metric_type: gauge
+  title: sf.org.apm.subscription.monitoringMetricSets
 
-sf.org.apm.numSpanBytesReceivedByToken:
-  brief: The number of bytes Splunk APM accepts for a specific access token
-    for a span after decompression after filtering and throttling.
+sf.org.apm.subscription.spanBytes:
+  brief: The entitlement for the number of bytes per minutes for your subscription plan.
   description: |
-    The number of bytes Splunk APM accepts for a specific access token
-    for a span after decompression, filtering, and throttling.
-
-      * Dimension(s): `orgId`, `tokenId`
-      * Data resolution: 1 second
-  metric_type: counter
-  title: sf.org.apm.numSpanBytesReceivedByToken
+    The entitlement for the number of bytes per minutes for your subscription plan.
+  
+        * Dimension(s): `orgId`
+        * Data resolution: 2 minutes
+  metric_type: gauge
+  title: sf.org.apm.subscription.spanBytes
 
-sf.org.apm.numAddSpansCalls:
-  brief: The number of calls to the `/v2/trace` endpoint.
+sf.org.apm.subscription.traces:
```

Comment:
style guide: spell out acronym TAPM on the first usage 👀

Reply:
Actually, Brian added that text. I don't know what TAPM is! I should have checked the entire file before I submitted.